tests/e2e_tests/multistep/test_incentive.py

```diff
@@ -0,0 +1,287 @@
+import asyncio
+import logging
+import sys
+import time
+
+import pytest
+
+import bittensor
+from bittensor.commands import (
+    RegisterCommand,
+    RegisterSubnetworkCommand,
+    StakeCommand,
+    RootRegisterCommand,
+    RootSetBoostCommand,
+)
+from tests.e2e_tests.utils import (
+    setup_wallet,
+    template_path,
+    repo_name,
+)
+
+logging.basicConfig(level=logging.INFO)
+
+
+@pytest.mark.asyncio
+async def test_incentive(local_chain):
```

Comment:
nitpick: I know you put comments across the test, but adding single-line docstring could looks elegant and helpful

Reply:
Good point! I added comments to all E2E tests I have added so far!